ideascube/tests/helpers.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+
+def migration_test(migrate_from, migrate_to):
```

Comment:
That decorator looks like a scary beast to maintain in the long run.

However, the API it provides (decorate test functions) is pretty great!